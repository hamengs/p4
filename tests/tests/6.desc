vmalloc single large page allocation
