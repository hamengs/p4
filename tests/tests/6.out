XV6_TEST_OUTPUT Start of program - base pages:3 huge pages:0
XV6_TEST_OUTPUT After vmalloc huge, 1MB - program base pages:3 huge pages:1
XV6_TEST_OUTPUT Sanity checking vmalloc address 1E000008
